docs/docs/dev_docs/contracts/portals/inbox.md

```diff
@@ -128,11 +145,26 @@ Computes the hash of a message.
 
 ```solidity
 function computeEntryKey(
-  DataStructures.L1ToL2Msg memory _message
-) external pure returns (bytes32 entryKey);
+  address _senderAddress,
+  uint256 _senderChainId,
+  bytes32 _recipientAddress,
+  uint256 _recipientVersion,
+  bytes32 _content,
+  bytes32 _secretHash,
+  uint32 _deadline,
+  uint64 _fee
+) external pure returns (bytes32);
 ```
 
 | Name           | Type        | Description |
 | -------------- | -------     | ----------- |
-| `_message`     | `L1ToL2Msg` | The message to compute hash for |
+| `_senderAddress`     | `address` | The address of the sender on L1. |
+| `_senderChainId`     | `uint256` | The chain id of the L1 blockchain. |
+| `_recipientAddress`     | `bytes32` | The aztec address of the recipient. |
+| `_recipientVersion`     | `uint256` | The aztec version of the recipient. |
+| `_content`        | `field` (~254 bits) | The content of the message. This is the data that will be passed to the recipient. The content is limited to be a single field for rollup purposes. If the content is small enough it can just be passed along, otherwise it should be hashed and the hash passed along (you can use our [`Hash`](https://github.com/AztecProtocol/aztec-packages/blob/master/l1-contracts/src/core/libraries/Hash.sol) utilities with `sha256ToField` functions)  |
+| `_secretHash`    | `field` (~254 bits)  | A hash of a secret that is used when consuming the message on L2. Keep this preimage a secret to make the consumption private. To consume the message the caller must know the pre-image (the value that was hashed) - so make sure your app keeps track of the pre-images! Use the [`computeMessageSecretHash`](https://github.com/AztecProtocol/aztec-packages/blob/master/yarn-project/aztec.js/src/utils/secrets.ts) to compute it from a secret. |
+| `_deadline`       | `uint256` | The message consumption deadline. If the message have not been removed from the `Inbox` and included in a rollup block by this point, it can be *cancelled* by the portal (the portal must implement logic to cancel). |
```

Comment:
```suggestion
| `_deadline`       | `uint256` | The deadline to consume the message by. If the message has not been removed from the `Inbox` and included in a rollup block by this time, it can be *cancelled* by the portal (the portal must implement logic to cancel). |
```